src/Impostor.Api/Net/MessageWriterExtensions.cs

```diff
@@ -0,0 +1,32 @@
+using System.Numerics;
+using Impostor.Api.Games;
+using Impostor.Api.Net.Inner;
+using Impostor.Api.Unity;
+
+namespace Impostor.Api.Net;
+
+public static class MessageWriterExtensions
+{
+    public static void Serialize(this GameCode gameCode, IMessageWriter writer)
+    {
+        writer.Write(gameCode.Value);
+    }
+
+    public static void Write(this IMessageWriter writer, IInnerNetObject? innerNetObject)
```

Comment:
Why allow calling with a null IInnerNetObject?

Reply:
To mimic base game behavior